lib/select2-3.5.1/select2.css

```diff
@@ -241,26 +243,15 @@ html[dir="rtl"] .select2-container .select2-choice .select2-arrow b {
 
 .select2-search input {
     width: 100%;
-    height: auto !important;
-    min-height: 26px;
     padding: 4px 20px 4px 5px;
     margin: 0;
-
+    font-size: 13px;
+    letter-spacing: 0.34px;
+    font-family: 'Open Sans', sans-serif;
```

Comment:
Is this font going to be used everywhere on the site? Seems odd to set it on such a specific element and not overall.